scripts/mods/botania.zs

```diff
@@ -439,3 +439,72 @@ recipes.removeByRecipeName('botania:tornadorod');   recipes.addShaped(<botania:t
 recipes.removeByRecipeName('botania:waterrod');     recipes.addShaped(<botania:waterrod>, [[null, <minecraft:potion>.withTag({Potion: 'minecraft:water'})], [<ore:livingwoodTwig>, null]]);
 recipes.removeByRecipeName('botania:skydirtrod');   recipes.addShapeless(<botania:skydirtrod>, [<botania:dirtrod>, <ore:elvenPixieDust>]);
 // ////////////////////////////////////////////////////////////////
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "campanimia"}), [
+<botania:rune:1>,
+<botania:rune:2>,
+<botania:rune:3>,
+<botania:rune>,
+<quark:rune:14>,
+<quark:rune:2>,
+<quark:rune:4>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "nuclianthus"}), [
+<quark:rune:5>,
+<quark:rune:4>,
+<quark:rune:4>,
+<quark:rune:1>,
+<botania:manaresource:1>,
+<botania:rune:6>,
+<botania:rune:6>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "rokku_eryngium"}), [
+<botania:manaresource:5>,
+<botania:manaresource:9>,
+<botania:rune:11>,
+<botania:rune:12>,
+<quark:rune:11>,
+<quark:rune>,
+<quark:rune>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "jikanacea"}), [
+<astralsorcery:itemtunedcelestialcrystal>.withTag({astralsorcery: {constellationName: "astralsorcery.constellation.horologium", crystalProperties: {collectiveCapability: 100, size: 900, fract: 0, purity: 100}}}),
+<botania:rune:14>,
+<botania:rune:9>,
+<quark:rune:2>,
+<quark:rune:4>,
+<quark:rune:4>,
+<thaumicaugmentation:material:5>,
+]);
+
+mods.botania.Apothecary.addRecipe(<botania:specialflower>.withTag({type: "echinacenko"}), [
+<quark:rune:2>,
+<quark:rune:2>,
+<quark:rune:5>,
+<botania:rune:10>,
+<botania:rune:12>,
+<botania:manaresource:8>,
+]);
```

Comment:
Must be cheaper and Terrasteel-level gated (instead of Elfheim gated) )since require you to stay near and not working instantly.